app/src/main/java/com/eventyay/organizer/data/faq/FaqDao.java

```diff
@@ -0,0 +1,23 @@
+package com.eventyay.organizer.data.faq;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import java.util.List;
+
+import io.reactivex.Observable;
+
+@Dao
+public interface FaqDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    void insertFaq(Faq faq);
+
+    @Query("DELETE FROM Faq WHERE id = :id")
+    void deleteFaq(long id);
```

Comment:
Change void to completable

Reply:
But `DELETE` queries can return only `void` and `int`?

From the [documentation](https://developer.android.com/reference/android/arch/persistence/room/Query):

`UPDATE or DELETE queries can return void or int. If it is an int, the value is the number of rows affected by this query.`

Reply:
No, it can handle Completable

Reply:
Okay :+1:

Reply:
Updated accordingly but DELETE query isn't working.